### ISSUE TITLE
`keep-options-order` warns on `greedy: true`

### ISSUE THREAD

**Issue by AlexandrHoroshih** [effector/eslint-plugin#107]
Effector's sample has special setting - `greedy: true`
It is needed in extremly rare cases, but sometimes it does

Current behavior:

For code like this
```ts
sample({
 greedy: true // disables default sync batching
 clock: eventHappened,
})
```
I get warning like this:
<img width="588" alt="image" src="https://user-images.githubusercontent.com/32790736/174777596-1eb4a0d0-fcb5-46ec-9905-d031b15fcd17.png">

Expected behavior:
`keep-options-order` either ignores `greedy` or requires explicit position for it (i would prefer to place it on very top of config)
